site/docs/v1/tech/oauth/integrations/cockroachdb.adoc

```diff
@@ -0,0 +1,59 @@
+---
+layout: doc
+title: "OAuth Integrations: CockroachDB"
+description: A practical approach of using FusionAuth SSO capabilities to log in to your Cockroach DB Console
+navcategory: login-methods
+---
+
+== Overview
+
+link:https://www.cockroachlabs.com/[CockroachDB] is a distributed SQL database designed for speed and scale. It offers a self-hosted deployment model, where you run the clusters in your own environment, and two SaaS options: dedicated and serverless. We'll use the dedicated version to set up Single Sign-On, allowing users to log in to your CockroachDB Console interface using their FusionAuth credentials.
```

Comment:
No need for links that have `http` in front of them to have the `link:` prefix. Please remove.